python/src/iceberg/core/util/bin_packing.py

```diff
@@ -0,0 +1,81 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+class PackingIterator:
```

Comment:
Part of the issue with starting the new project structure in the same PR as relocating the old one is that we're already establishing some of the same packaging structure, which I don't think we want to do just yet.

Reply:
Totally agree. I did that to kind of avoiding changing github build script. Given that we need to redesign the python library structure, I will update the github build script and remove the 2nd commit.